src/handlers/kubectl/K8sDeleteObjectActionHandler.ts

```diff
@@ -45,6 +40,18 @@ export class K8sDeleteObjectActionHandler extends ActionHandler {
         snapshot: ActionSnapshot,
```

Comment:
@vlad-tkachenko why we declare unused parameters?

Reply:
Just a method signature derived from the base class, nothing more.